### ISSUE TITLE
Error: no writecb in Transform class

### ISSUE THREAD

**Issue by Rudloff** [plus3network/gulp-less#207]
I get this error when running gulp-less with node 0.10 (the version packaged in Debian stable):

```
Potentially unhandled rejection [2] Error: no writecb in Transform class
    at afterTransform (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:75:33)
    at TransformState.afterTransform (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:59:12)
    at /home/pierre/www/KNACSS/node_modules/gulp-less/index.js:58:14
    at tryCatchReject (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/makePromise.js:845:30)
    at runContinuation1 (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/makePromise.js:804:4)
    at Rejected.when (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/makePromise.js:625:4)
    at Pending.run (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/makePromise.js:483:13)
    at Scheduler._drain (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/Scheduler.js:62:19)
    at Scheduler.drain (/home/pierre/www/KNACSS/node_modules/gulp-less/node_modules/accord/node_modules/when/lib/Scheduler.js:27:9)
    at process._tickCallback (node.js:419:13)
```

**Comment by contra** [comment]
Can you use the latest version of node and run `npm update`?

**Comment by Rudloff** [comment]
I confirm it works correctly if I install node 0.12 from NodeSource, but I would prefer to stick with the version packaged in the stable repositories.

**Comment by contra** [comment]
@Rudloff Node 4.0 is the latest stable (and LTS) release - 0.10 is over a year old and no longer considered stable, 0.12 is also pretty old.

**Comment by sakshigandhi** [comment]
I am facing the same issue, and I cannot upgrade from v0.10 to v0.12

**Comment by ArjanKw** [comment]
Getting the same error when trying to compile less to css. Updated to latest version node/npm to no avail. Cleaned cache (npm cache clean -f).

$ node -v
v4.2.4
$ npm -v
3.5.2

[23:33:57] Using gulpfile c:\wamp\www\gulpfile.js
[23:33:57] Starting 'default'...
[23:33:57] Finished 'default' after 13 ms
[23:34:01] Starting 'less'...
[23:34:01] Finished 'less' after 8.87 ms
Potentially unhandled rejection [2] Error: no writecb in Transform class
    at afterTransform (c:\wamp\www\node_modules\gulp-less\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:75:33)
    at TransformState.afterTransform (c:\wamp\www\node_modules\gulp-less\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:59:12)
    at c:\wamp\www\node_modules\gulp-less\index.js:58:14
    at tryCatchReject (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\makePromise.js:845:30)
    at runContinuation1 (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\makePromise.js:804:4)
    at Rejected.when (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\makePromise.js:625:4)
    at Pending.run (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\makePromise.js:483:13)
    at Scheduler._drain (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\Scheduler.js:62:19)
    at Scheduler.drain (c:\wamp\www\node_modules\gulp-less\node_modules\accord\node_modules\when\lib\Scheduler.js:27:9)
    at doNTCallback0 (node.js:419:9)

**Comment by ArjanKw** [comment]
Tried the same today and it worked, have no clue what has changed.

**Comment by sam2x** [comment]
Got the same error, tried all advices (cleaning cache, updating), and it's not working, still got this error : 3

     Potentially unhandled rejection [2] Error: no writecb in Transform class

My versions : 

     $ node -v
     v5.4.1
     $ npm -v
     3.3.12
     $ gulp -v
     [16:14:22] CLI version 3.9.0
     [16:14:22] Local version 3.9.1

Please help, would listen any comments (it's an urgent task, since it broke my whole dev process :/)

**Comment by sam2x** [comment]
Tried with all versions of gulp-less > 3.0.0 and the master version. Not working.

**Comment by sam2x** [comment]
Downgrading gulp-less to "^1.3.2" worked -_-

**Comment by Saturate** [comment]
Maybe this node engines should be changed in the `package.json` to reflect what version is working/needed?

https://github.com/plus3network/gulp-less/blob/master/package.json#L13-L15 
https://docs.npmjs.com/files/package.json#engines

**Comment by licx** [comment]
Any news on this issue?

**Comment by wesleypimentel** [comment]
I solved my problem with "in writecb in Transform class" in this way:

I searched file by file for possible syntax errors or something.
I noticed that was related with a imported file with some mixins and animations.
Then checked the animation keyframes:

@keyframes "foo" { ... ("with quotes")

I decided remove the quotes, like this:

@keyframes foo { ... (without quotes)

After this adjustment the gulp-less was compiled perfectly.
